README.md

```diff
@@ -190,15 +191,21 @@ SD + APP      | Yes       | **See notes 1 and 2 below**
 was added in nrfutil 3.1.0 and is required since 3.2.0 in case the package should contain SD (+ BL) + APP. Also, since version 3.2.0 the new ID is copied to `--sd-req` list so that
 in case of a link loss during APP update the DFU process can be restarted. In that case the new SD would overwrite itself, so `--sd-req` must contain also the ID of the new SD.
 
-Boot validation for a SD or APP update can be activated by setting the `--sd-boot-validation` or '--app-boot-validation' to the preferred validation method. When boot validation is set, the bootloader will store the signature of the SD or APP in the bootloader setting page and verify the signature with the firmware in flash on every boot. This is only supported by bootloaders using bootloader settings version 2, see 'settings' for more information.
+**Note 3:** When creating update packages of bootloaders compiled from nRF5 SDK 15.3.0 and higher, nrfutil version 5.0.0 must be used. This is because of changes to the bootloader projects in the nRF5 SDK, and if an old nrfutil version is used the size of the generated packages
+will be too large.
+
+Boot validation for a SD or APP update can be activated by setting the `--sd-boot-validation` or `--app-boot-validation` to the preferred
+validation method. When boot validation is set, the bootloader will store the signature of the SD or APP in the bootloader setting page
```

Comment:
A signature will be stored only if the boot validation type is SECP256. For SHA256, a hash is stored, and for CRC, a checksum is stored.

Reply:
Is there a "common" name for signature, hash and checksum?

Reply:
Validation data.

"When boot validation is set, the bootloader will store the validation data (signature, hash, or checksum) of the SD or APP in the bootloader setting page ..." ?